templater/jobs/types/types.go

```diff
@@ -68,4 +68,5 @@ type JobConfig struct {
 	AutomountServiceAccountToken string         `json:"automountServiceAccountToken,omitempty"`
 	Cluster                      string         `json:"cluster,omitempty"`
 	SkipBuildkitCheck            bool           `json:"skipBuildkitCheck,omitempty"`
+	ProjectPath                  string         `json:"PROJECT_PATH,omitempty"`
```

Comment:
```suggestion
	ProjectPath                  string         `json:"projectPath,omitempty"`
```
You need to add a projectPath field in all the minimal configs above, only then will it get put into the JobConfig struct when unmarshaling the YAML.